index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Beyonce Knowles Resume</title>
+  <link rel="stylesheet" type="text/css" href="css/styles.css">
+</head>
+<body>
+
+  <div>
+    <img src="Images/BeyonceFlower.jpg" alt="Beyonce with a flower">
```

Comment:
the image is not loading because you are calling `Images/BeyonceFlower.jpg` however your folder structure is `images/BeyonceFlower.jpg` 🔨